ihmc-sensor-processing/src/main/java/us/ihmc/sensorProcessing/heightMap/HeightMapCell.java

```diff
@@ -0,0 +1,103 @@
+package us.ihmc.sensorProcessing.heightMap;
+
+import com.google.common.util.concurrent.AtomicDouble;
+import gnu.trove.list.array.TDoubleArrayList;
+import gnu.trove.list.array.TIntArrayList;
+import us.ihmc.euclid.tools.EuclidCoreTools;
+import us.ihmc.yoVariables.providers.IntegerProvider;
+import us.ihmc.yoVariables.providers.LongProvider;
+
+/**
+ * Contains data for a given XY grid cell.
+ */
+class HeightMapCell
```

Comment:
This class represents a single cell in the HeightMap and contains all the logic for updating it.